Dependencies taffydb and underscore missing in package.json

I installed jsdoc via: 

```
npm install jaguarjs-jsdoc
```

however when I tried to create a documentation with it I got: 

```
Cannot find module 'underscore'
Cannot find module 'taffydb'
```

Running `npm install underscore taffydb` fixed the issue, but those dependencies should really be in the package.json.

--- a/package.json
+++ b/package.json
@@ -4,6 +4,8 @@
   "description": "Jaguar.js template for JSDoc 3",
   "main": "Gruntfile.js",
   "dependencies": {
+    "taffydb": "^2.7.2",
+    "underscore": "^1.7.0"
   },
   "devDependencies": {
     "connect-livereload": "~0.3.2",

